### PR TITLE
4055 2

### PR DESCRIPTION
**Pull request by DariaElagina**
For the first four texts no folia are provided by the catalogue.

### CODE REVIEW

File: StPetersburg/InEtn40552.xml [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+                  </msItem>
+                  <msItem xml:id="ms_i2">
+                    <title type="complete" ref="LIT2993RepCh274" cert="medium"></title>
+                    <textLang mainLang="gez"></textLang>
+                    <incipit xml:lang="gez">በስመ፡ እግዚአብሔር፡ ቀዳማዊ፡ ዘእንበለ፡ ትማልም፡</incipit>
+                  </msItem>
+                  <msItem xml:id="ms_i3">
+                    <title type="complete" ref="LIT4963Naahabeya"></title>
+                    <textLang mainLang="gez"></textLang>
+                    <incipit xml:lang="gez">ነዓ፡ ኀቤየ፡ ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡</incipit>
```
Comment (thea-m) [inline StPetersburg/InEtn40552.xml]: አ or ኦ?

File: StPetersburg/InEtn40552.xml [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                    <textLang mainLang="gez"></textLang>
+                    <incipit xml:lang="gez">ወይእዜኒ፡ ንስአሎ፡ ከመ፡ ይስረይ፡ ለነ፡ ኃጣውኢነ፡</incipit>
+                    <note>The name of the owner <persName ref="PRS12876HaylaS" role="owner"></persName> has been added with another hand at the end of the text.</note>
+                  </msItem>
+                  <msItem xml:id="ms_i5">
+                    <locus from="5r" to="8v"/>
+                    <title type="incomplete" ref="LIT2444Temher"></title>
+                    <textLang mainLang="gez"></textLang>
+                    <note>The end of the text is missing.</note>
+                    <explicit xml:lang="gez">አኮ፡ በዝንቱ፡ ከናፍር፡ ውዱዳት፡ ወአኮ፡ በልሳነ፡ ሥጋ፡ ዘባቲ፡ ድድቅ፡ ወሐሰት፡ ይወጽእ፡</explicit>
```
Comment (thea-m) [inline StPetersburg/InEtn40552.xml]: ጽድቅ?

File: StPetersburg/InEtn40552.xml (hunk 1 above)
Comment (DariaElagina) [reply StPetersburg/InEtn40552.xml]: ኦ

File: StPetersburg/InEtn40552.xml (hunk 2 above)
Comment (DariaElagina) [reply StPetersburg/InEtn40552.xml]: ድድቅ፡ - accident

**Pull request merged**